assets/prototype/domain/eventEditor/data/types.ts

```diff
@@ -4,7 +4,10 @@ export interface Entity {
 	name?: string;
 	__typename?: string;
 }
-export interface EntityEgde {
+
+export type EntityIds = string[];
```

Comment:
I think `type EntityId = string;` would be better because we can use it as `EntityId` as well as `EntityId[]`

Reply:
Updated: https://github.com/eventespresso/event-espresso-core/pull/2076/commits/c7e2b27ddfa96a960f38b76672b0793c753e2557